features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/filters/RoomListFilter.kt

```diff
@@ -26,13 +26,15 @@ enum class RoomListFilter(val stringResource: Int) {
     Unread(R.string.screen_roomlist_filter_unreads),
     People(R.string.screen_roomlist_filter_people),
     Rooms(R.string.screen_roomlist_filter_rooms),
-    Favourites(R.string.screen_roomlist_filter_favourites);
+    Favourites(R.string.screen_roomlist_filter_favourites),
+    Invites(R.string.screen_roomlist_filter_invites);
 
-    val oppositeFilter: RoomListFilter?
+    val incompatibleFilters: Set<RoomListFilter>
         get() = when (this) {
-            Rooms -> People
-            People -> Rooms
-            Unread -> null
-            Favourites -> null
+            Rooms -> setOf(People, Invites)
+            People -> setOf(Rooms, Invites)
+            Unread -> setOf(Invites)
```

Comment:
Not sure, but we may want to have "unread" invites. Also the green dot on invite item in room list should be hidden when the invite has been visited.

Reply:
This has been discussed, and for now we don't want